.github/needs_more_info.yml

```diff
@@ -1,6 +1,6 @@
 {
   daysUntilClose: 7,
-  needsMoreInfoLabel: 'more info',
-  perform: false,
-  closeComment: "This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
+  needsMoreInfoLabel: 'needs repro',
+  perform: true,
+  closeComment: "This issue has been closed automatically because it needs more information and has not had recent activity in the last 7 days. If you have more info to help resolve the issue, leave a comment"
```

Comment:
should the message say that it needs repro rather than more info...?

Reply:
Like I stated in the PR I think we should just replace `needs repro` with `needs info` since saying something needs repro is basically saying something needs more info.